datafusion/sqllogictest/test_files/explain.slt

```diff
@@ -411,3 +411,40 @@ logical_plan
 physical_plan
 01)ProjectionExec: expr=[{c0:1,c1:2.3,c2:abc} as struct(Int64(1),Float64(2.3),Utf8("abc"))]
 02)--PlaceholderRowExec
+
+
+
+
+query TT
```

Comment:
Any reason for not adding these as unit tests in the `expr_simplifier` module instead? 

They feel quite output place in file called `explain.slt`.

Reply:
Sure